precommit.py

```diff
@@ -30,25 +30,28 @@ def main() -> int:
     if overwrite:
         subprocess.check_call([
             "yapf", "--in-place", "--style=style.yapf", "--recursive", "tests",
-            "gswrap", "setup.py", "precommit.py"
+            "gswrap", "benchmark", "setup.py", "precommit.py"
```

Comment:
I think there is no need to run benchmarks in precommit, let's ask Marko what he thinks